usr/src/man/man1m/zfs.1m

```diff
@@ -1091,9 +1091,11 @@ Quotas cannot be set on volumes, as the
 .Sy volsize
 property acts as an implicit quota.
 .It Sy logicalquota Ns = Ns Em size Ns | Ns Sy none
-This property have the same behaviour as quota, but on logical space used.
-When compression or deduplication is active, the space accounting is done on
-logical used property.
+This property have the same behaviour as quota, but on logical (i.e. uncompressed)
+space used, referred by
```

Comment:
`referred` -> `as reported by the` (to avoid confusion with "referenced" space / `logicalrefquota`)